javascript-modal/main.js

```diff
@@ -0,0 +1,22 @@
+var click = false;
+var overL = document.querySelector('.overlay');
+var pop = document.querySelector('.pop');
+var openB = document.querySelector('.open-button');
+var closeB = document.querySelector('.close-button');
+
+function popUp(event) {
+  if (click === false) {
+    overL.className = 'overlay';
+    pop.className = 'pop';
+    openB.className = 'open-button';
+    click = true;
+  } else {
+    overL.className = 'overlay-on';
+    pop.className = 'pop-on';
+    openB.className = 'open-button-off';
+    click = false;
+  }
```

Comment:
Rather than having one function that has to decide whether it should open or close the modal, I would make two separate functions, one to open it, and one to close it.